flake.nix

```diff
@@ -86,6 +86,8 @@
       gpd-win-2 = import ./gpd/win-2;
       gpd-win-max-2-2023 = import ./gpd/win-max-2/2023;
       hp-elitebook-2560p = import ./hp/elitebook/2560p;
+      hp-elitebook-830-g6 = import ./hp/elitebook/830/g6;
```

Comment:
Do we need both?

Reply:
I removed the one that didn't match the pattern of the adjacent imports